pyqtgraph/exporters/CSVExporter.py

```diff
@@ -3,22 +3,20 @@
 from .Exporter import Exporter
 from ..parametertree import Parameter
 from .. import PlotItem
-from ..python2_3 import asUnicode
-
-translate = QtCore.QCoreApplication.translate
+from ..python2_3 import asUnicode 
 
 __all__ = ['CSVExporter']
     
     
 class CSVExporter(Exporter):
     Name = "CSV from plot data"
-    windows = []
+    windows = [] 
     def __init__(self, item):
         Exporter.__init__(self, item)
         self.params = Parameter(name='params', type='group', children=[
-            {'name': 'separator', 'title': translate("Exporter", 'separator'), 'type': 'list', 'value': 'comma', 'values': ['comma', 'tab']},
-            {'name': 'precision', 'title': translate("Exporter", 'precision'), 'type': 'int', 'value': 10, 'limits': [0, None]},
-            {'name': 'columnMode', 'title': translate("Exporter", 'columnMode'), 'type': 'list', 'values': ['(x,y) per plot', '(x,y,y,y) for all plots']}
+            {'name': 'separator', 'type': 'list', 'value': 'comma', 'values': ['comma', 'tab']},
```

Comment:
for these sections can you re-add the sections `'title': translate("Exporter", 'separator')` bits that you can see if you look at the removed lines?  That code is what allows for i18n language translation compatability